### PR TITLE
change flag name from useUnifiedSchema into useCommonAlertSchema per …

### PR DESCRIPTION
**Pull request by kikicandy11**
…UX requirement

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/monitor/resource-manager/Microsoft.Insights/stable/2019-03-01/actionGroups_API.json
Comment (jianghaolu) [inline specification/monitor/resource-manager/Microsoft.Insights/stable/2019-03-01/actionGroups_API.json]: Are you guys changing a property name in the existing API version? Did ARM approve this breaking change?

Comment (kikicandy11) [reply specification/monitor/resource-manager/Microsoft.Insights/stable/2019-03-01/actionGroups_API.json]: Hi, we just commit the new api version on 3/1 and this is not use yet, UX guy just started working on the change.
UX raise the request to change the field name on 3/4 , so I raised this pr.

Comment (jianghaolu) [reply specification/monitor/resource-manager/Microsoft.Insights/stable/2019-03-01/actionGroups_API.json]: This change should be part of the spec for the new API version then. You cannot introduce breaking changes to a currently stable API version.

Comment (kikicandy11) [reply specification/monitor/resource-manager/Microsoft.Insights/stable/2019-03-01/actionGroups_API.json]: Hi, Then this apiversion 2019-03-01 will never be used. do youm mean create an 2019-03-06？The change is not in RP Prod yet. no one use it now

Comment (kikicandy11) [reply specification/monitor/resource-manager/Microsoft.Insights/stable/2019-03-01/actionGroups_API.json]: @jianghaolu  shall we involve the ARM team to review the get some suggestion? This is a urgent fix, we're waiting for approval for RP PROD deployment.

**Review by jianghaolu — APPROVED** [review]

**Comment by jianghaolu** [comment]
@ALL SDK OWNERS
As of 3/7/2019, the APIs are not in prod yet. Please confirm with @kikicandy11 before releasing any SDK. Thanks!

**Pull request merged**
